docs/docsite/rst/network/user_guide/platform_eos.rst

```diff
@@ -0,0 +1,129 @@
++---------------------------+-----------------------------------------------+-----------------------------------------+
+| | **Credentials**         | | uses SSH keys / SSH-agent if present        | | uses HTTPS certificates if present    |
+| |                         | | accepts ``-u myuser -k`` if using password  | |                                       |
++---------------------------+-----------------------------------------------+-----------------------------------------+
+| **Indirect Access**       | via a bastion (jump host)                     | via a web proxy                         |
++---------------------------+-----------------------------------------------+-----------------------------------------+
+| | **Connection Settings** | | ``ansible_connection: network_cli``         | | ``ansible_connection: local``         |
+| |                         | |                                             | | Requires ``transport: eapi``          |
+| |                         | |                                             | | in the ``provider`` dictionary        |
++---------------------------+-----------------------------------------------+-----------------------------------------+
+| | **Enable Mode**         | | supported - use ``ansible_become: yes``     | | supported - use ``authorize: yes``    |
+| | (Privilege Escalation)  | | with ``ansible_become_method: enable``      | | in the ``provider`` dictionary        |
++---------------------------+-----------------------------------------------+-----------------------------------------+
+| **Returned Data Format**  | ``stdout[0].``                                | ``stdout[0].messages[0].``              |
++---------------------------+-----------------------------------------------+-----------------------------------------+
+
+
+Using CLI in Ansible 2.5
+================================================================================
+
+Example CLI ``group_vars/eos.yml``
+----------------------------------
+
+.. code-block:: yaml
+
+   ansible_connection: network_cli
+   ansible_network_os: eos
+   ansible_user: myuser
+   ansible_ssh_pass: !vault...
+   ansible_become: yes
+   ansible_become_method: enable
+   ansible_ssh_common_args: '-o ProxyCommand="ssh -W %h:%p -q bastion01"'
+
+
+- If you are using SSH keys (including an ssh-agent) you can remove the ``ansible_ssh_pass`` configuration.
+- If you are accessing your host directly (not through a bastion/jump host) you can remove the ``ansible_ssh_common_args`` configuration.
+- If you are accessing your host through a bastion/jump host, you cannot include your SSH password in the ``ProxyCommand`` directive. To prevent secrets from leaking out (for example in ``ps`` output), SSH does not support providing passwords via environment variables.
+
+Example CLI Task
+----------------
+
+.. code-block:: yaml
+
+   - name: Backup switch (eos)
```

Comment:
The description can be updated as `Backup switch running configuration (eos)`.
Applicable at other places as well.

Reply:
I think this is applicable for other platforms as well.

Reply:
Thanks for the feedback. I went with "Backup current switch configuration" to make it clear that "switch" is a noun here, not a verb. Addressed in d0d7d916162a3f8b6e5a6b24b2d2ab715124526a